src/client/components/dashboard/CreateNewModal.tsx

```diff
@@ -77,7 +86,7 @@ export const CreateNewModal: FC<CreateNewModalProps> = ({
     description: string
   }) => {
     createChecker.mutate({
-      ...(checker || initial),
+      ...(checker ? { ...checker, id: uuidv4() } : initial),
```

Comment:
Actually why not...
```suggestion
      ...(checker || initial),
      id: uuidv4()
```

Then eliminate the `id` field in `initial`

Reply:
Makes sense! Let me update that.